### PR TITLE
rename entity.paused to entity.isPlaying (fixes #1044)

### LINKED ISSUE #1044: [code quality] rename entity `paused` to entity `isPlaying`
`paused` is a negative/off state, `isPlaying` would be more appropriate to keep track of

### PR DESCRIPTION
**Pull request by ngokevin**
Want to do this before 0.2.0 because this is public API which will be hard to undo

### CODE REVIEW

File: src/core/a-entity.js [hunk 1]
```diff
@@ -14,9 +14,6 @@ var AEntity;
 
 /**
  * Entity element definition.
- * Entities represent all elements that are part of the scene, and always have
```
Comment (dmarcos) [inline src/core/a-entity.js]: Why is this comment gone?

Comment (ngokevin) [inline src/core/a-entity.js]: not necessary to have tutorial material here since it's explained in the docs

Comment (ngokevin) [inline src/core/a-entity.js]: i can put it back though, this one isn't too bad

Comment (ngokevin) [inline src/core/a-entity.js]: ok i put it back, but tightened it a little

**Pull request merged**